test/dynamo/test_fallback.py

```diff
@@ -0,0 +1,78 @@
+import torch
+import torch_xla
+import torch_xla.core.xla_model as xm
+import torch._dynamo as dynamo
+import torch._dynamo.test_case as dt
+
+
+class XlaDynamoFallbackTest(dt.TestCase):
+
+  def test_operator_fallback(self):
+
+    def fn_fallback(M, mat1, mat2):
+      A = torch.cummin(M, 1)
+      B = torch.add(A[0], mat1)
+      return torch.add(B, mat2)
+
+    dynamo_fn = torch.compile(fn_fallback, backend="torchxla_trace_once")
+    M = torch.randn(5, 10)
+    mat1 = torch.randn(5, 10)
+    mat2 = torch.randn(5, 10)
+    xla_M = M.to(xm.xla_device())
+    xla_mat1 = mat1.to(xm.xla_device())
+    xla_mat2 = mat2.to(xm.xla_device())
+
+    cpu_res = fn_fallback(M, mat1, mat2)
+    xla_res = dynamo_fn(xla_M, xla_mat1, xla_mat2)
+
+    self.assertTrue(torch.allclose(cpu_res, xla_res.cpu()))
+
+  def test_operator_operand_fallback(self):
+
+    def fn_fallback(M, mat1, mat2, beta):
+      # xla currently only support alpha and beta == 1
+      A = torch.addmm(M, mat1, mat2, beta=beta)
+      B = torch.add(A, mat1)
+      return torch.add(B, mat1)
+
+    dynamo_fn = torch.compile(fn_fallback, backend="torchxla_trace_once")
+    M = torch.randn(2, 3)
+    mat1 = torch.randn(2, 3)
+    mat2 = torch.randn(3, 3)
+    xla_M = M.to(xm.xla_device())
+    xla_mat1 = mat1.to(xm.xla_device())
+    xla_mat2 = mat2.to(xm.xla_device())
+
+    cpu_res = fn_fallback(M, mat1, mat2, 0.5)
+    xla_res = dynamo_fn(M, mat1, mat2, 0.5)
+
+    self.assertTrue(torch.allclose(cpu_res, xla_res.cpu()))
```

Comment:
ditto, counter check for compilation, execution. I think we also want to check that no `aten::` counter is accumulated. Let me keep reading and figure how is the fallback op being executed.

Reply:
After reading through the pr, it is still unclear to me how fallback op is being handled. Will be execute on CPU on the pytorch end? or it will be execute as lazy and go through our fallback op handling logic?

Reply:
To verify, we do see the `xla::_to_cpu` counter.

Reply:
I think that also explains why the `CompileTime` metric increases by one when seeing a fallback op.

Reply:
To clarify:
* `CompileTime` increases by one for unsupported op when testing if an op goes through CPU fallback or not
* `ExecuteTime` increases
  * Whenever an unsupported op is executed through lazy CPU fallback (e.g., in `FallBackNodeCollector` and `InputCollector`)
  * Whenever a compiled subgraph is executed
But with these, the final metric numbers still do not match. I'm still looking into the problem.

Reply:
Yeah, we're executing as lazy and going through our fallback op handling logic. We should see the `aten::` metrics when we execute the unsupported ops. 

@seanlatias, which final metric numbers do not match, the metric numbers in the fallback unit tests?

Reply:
For the `test_operator_fallback`, I see `CompileTime` to be 2, which makes sense. However, with `ExecuteTime`, it is 5. I still couldn't explain.

Reply:
I was playing a bit with the unit test case, and making it into a simpler test case. For what it's worth, with a simpler unit test like below:
```
  def test_operator_fallback(self):

    def fn_fallback(t):
      # As of 05/18/2023, torch.median is not lowered by PyTorch/XLA
      return torch.median(t)

    torch._dynamo.reset()
    met.clear_counters()
    device = xm.xla_device()

    dynamo_fn = torch.compile(fn_fallback, backend="torchxla_trace_once")
    t = torch.randn(5)
    t_xla = t.to(device)

    cpu_res = fn_fallback(t)
    xla_res = dynamo_fn(t_xla)

    print('CompileTime:', met.metric_data('CompileTime')[0])
    print('ExecuteTime:', met.metric_data('ExecuteTime')[0])

    self.assertTrue(torch.allclose(cpu_res, xla_res.cpu()))
```
I was able to see:
```
CompileTime: 2
ExecuteTime: 2
```
Let me look into the existing `test_operator_fallback` with  the `cummin` op.